lib/requirements_config.yaml

```diff
@@ -0,0 +1,12 @@
+test_kit_id: bulk-data-test-kit
+
+suites: 
+  - id: bulk_data_test_kit
+    class_name: BulkDataTestKit::BulkDataV200::BulkDataTestSuite
+    suite_actor: Provider
```

Comment:
Actors in the bulk data requirements are Client and Server, so this suite actor needs to be one of those. These tests are testing the Server, so use that here.